src/backend/src/app.js

```diff
@@ -56,49 +50,27 @@ process.on('exit', (code) => {
  * @param {string} [options.version] Device Version
  * @param {Buffer} options.rootKey
  * @param {string} options.migrationsFolderPath
- * @param {string} options.sharedStoragePath Path to app-specific external file storage folder
```

Comment:
we no longer need this option because we're strictly going to use the app's "internal" storage - AKA the document directory - to store the custom maps. The path to this directory is already available via nodejs-mobile e.g. `const privateStorageDir = rnBridge.app.datadir()`

The setup for this will be done in the other PR